lib/normalize_title_filter.js

```diff
@@ -29,6 +29,7 @@ module.exports = (hyper, req, next, options, specInfo) => {
             }
         });
     }
+    rp.title = rp[options.title] ? rp[options.title] : rp.title;
```

Comment:
1. rp[options.title] || rp.title is nicer
2. if options.title is undefined we'd be in trouble. so we need to go
(options.title && rp[options.title]) || rp.title the brackets are not really needed, but it's easier to read without thinking.